.circleci/config.yml

```diff
@@ -968,36 +993,40 @@ workflows:
               - << pipeline.trigger_source >>
         - not: << pipeline.parameters.manual >>
     jobs:
-      - test:
+      #
+      # Linting
+      #
+      - tox-base:
           name: "code-check"
-          image: "python:3.8"
-          toxenv: "protocheck,generatecheck,codecovcheck,mypy,mypy-report,pyupgrade,black,flake8,docstrings"
```

Comment:
You dropped codecovcheck here... please add it back and merge this into your branch:
https://github.com/wandb/wandb/pull/4076

Reply:
oops... will do...